MarkdownConverter/Converter/MarkdownSourceConverter.cs

```diff
@@ -172,11 +171,31 @@ IEnumerable<OpenXmlCompositeElement> Paragraph2Paragraphs(MarkdownParagraph md)
                 var runs = new List<Run>();
                 var onFirstLine = true;
                 IEnumerable<ColorizedLine> lines;
-                if (lang == "csharp" || lang == "c#" || lang == "cs") lines = Colorizer.Colorize("cs", code, Colorize.CSharp);
-                else if (lang == "vb" || lang == "vbnet" || lang == "vb.net") lines = Colorizer.Colorize("vb", code, Colorize.VB);
-                else if (lang == "" || lang == "xml") lines = Colorizer.Colorize("plain", code, Colorize.PlainText);
-                else if (lang == "antlr") lines = Colorizer.Colorize("antlr", code, Antlr.ColorizeAntlr);
-                else { Report.Error("MD09", $"unrecognized language {lang}"); lines = Colorize.PlainText(code); }
+                switch (lang)
```

Comment:
Think how much nicer this would look in C# 8 😄:

```csharp
lines = lang switch
{
“csharp” => Colorize.CSharp(code),
“c#” => Colorize.CSharp(code),
“cs” => Colorize.CSharp(code),
“vb” => Colorize.VB(code),
// ….
}
```

Reply:
Indeed, I think switch expressions are the part of C# 8 I'll look forward to most here - but I still think it's worth waiting until it's not in preview, just for simplicity. Hopefully not long :)